servicetalk-transport-netty/src/main/java/io/servicetalk/transport/netty/NettyIoExecutors.java

```diff
@@ -35,13 +36,30 @@ private NettyIoExecutors() {
      *
      * @param ioThreads number of threads.
      * @param threadFactory the {@link ThreadFactory} to use. If possible you should use an instance
-     * of {@link IoThreadFactory} as it allows internal optimizations.
+     * of {@link NettyIoThreadFactory} as it allows internal optimizations.
```

Comment:
Public module should not reference internal classes.

Reply:
I have removed this reference in the public module.